docker-compose.yaml

```diff
@@ -3,14 +3,15 @@ version: '2'
 services:
   app:
     build: ./
+    image: opencart-2.2.x
     restart: always
     ports:
       - 8080:80
     environment:
       OC_ROOT_DIR: /var/www/html
     volumes:
       - app:/var/www/html
```

Comment:
I think these volumes should be prefixed with OpenCart version. `opencart_2_2_app:/var/www/html` 
I ran into an issue where I think volume was overwritten when changing from version 1.5.x to version 2.2.x because the folder name is now same and volumes become `smaily-opencart-module_app` and `smaily-opencart-module_database` for all versions.

Reply:
Other option would be to use Docker Compose's functionality - `docker-compose -p oc22x up`. With this volumes will be prefixed by correct name. Granted this is hard to remember all the time, but I could create a command line interface for that. Or we could just leave it up for the developer to manage this?

Reply:
From our plugins, only Magento uses `tusk` because there are many tasks to automate or to reduce typing. The current workflow for the rest of the plugins is quite nice - run only `docker-compose up -d` and you have your containers up. I think both options would work. `tusk up` is also quite fast and intuitive.

The manual prefix option would mean that whenever you get back to starting your containers you would have to look for what prefix did you use last time, so I wouldn't go with that option. As we update these plugins once every 3-4 months I would keep everything regarding setting up the environment quite similar across all integrations.

Reply:
Fair enough. I will name volumes `app-2.2.x` and `database-2.2.x`, this way the name will be something like `smaily-opencart-module_app-2.2.x`.